content/vendor/guides/configuring-application-snapshots.md

```diff
@@ -0,0 +1,345 @@
+        env:
+        - name: POSTGRES_USER
+          value: "postgres"
+          # Required for pg_isready in the health probes....
+        - name: PGUSER
+          value: "postgres"
+        - name: POSTGRES_DB
+          value: "postgres"
+        - name: POSTGRES_INITDB_ARGS
+          value: ""
+        - name: PGDATA
+          value: /var/lib/postgresql/data/pgdata
+        - name: POSTGRES_PASSWORD
+          value: "postgres"
+        - name: POD_IP
+          valueFrom: { fieldRef: { fieldPath: status.podIP } }
+        ports:
+        - name: postgresql
+          containerPort: 5432
+        livenessProbe:
+          exec:
+            command:
+            - sh
+            - -c
+            - exec pg_isready --host $POD_IP
+          initialDelaySeconds: 60
+          timeoutSeconds: 5
+          failureThreshold: 6
+        readinessProbe:
+          exec:
+            command:
+            - sh
+            - -c
+            - exec pg_isready --host $POD_IP
+          initialDelaySeconds: 5
+          timeoutSeconds: 3
+          periodSeconds: 5
+        resources:
+          requests:
+            cpu: 100m
+            memory: 256Mi
+
+        volumeMounts:
+        - name: postgresql-vct
+          mountPath: /var/lib/postgresql/data/pgdata
+          subPath: postgresql-db
+        - name: postgresql-backup
+          mountPath: /backup
+          subPath: postgresql-db
+  volumeClaimTemplates:
+  - metadata:
+      name: postgresql-vct
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 2Gi
+  - metadata:
+      name: postgresql-backup
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 2Gi
+
+```
+{{< notes title="Note" >}}
+Note that in the command in the label annotation we are backing up the appdirectdb database, which does not exist when the application is first deployed. 
+For testing purposes, you could change it to back up the postgres database.
+{{< /notes >}}
+### Configuring the Cluster
+
+KOTS uses the [Velero](https://velero.io/) open source project to provide back up capabilities and will need to be added to the cluster. 
+The process for adding Velero will depend if the cluster is being managed by KOTS or not.
+
+#### Clusters Managed by KOTS
+
+For those applications running on a cluster [managed by KOTS](https://kots.io/vendor/packaging/embedded-kubernetes/), add the Velero add-on to the Kubernetes KURL installer. 
+While there are various configuration options as outlined [here](https://kurl.sh/docs/add-ons/velero), you should only need to add the following to your KURL Kubernetes installer.
+
+```shell
+  velero:
+    version: latest
+```
+
+Make sure to promote the installer to the same channel as the application.
+
+#### Other Clusters
+
+Since KOTS does not manage the cluster in this instance, it has to be done by the end user and covered in [Part II](#check-velero-installation).
```

Comment:
I'm always a fan of avoiding `it` and `this` when we can drop in a more clear instance of what we're referring to.

```suggestion
Since KOTS does not manage the cluster in this instance, the Velero installation has to be done by the end user and covered in [Part II](#check-velero-installation).
```

Reply:
Also I'm not sure how the heading you link to explains how to install velero in an existing cluster scenario, but I'm gonna shut up and keep reading for now 🙂